flink-runtime-web/web-dashboard/src/app/share/common/dagre/graph.ts

```diff
@@ -40,10 +40,12 @@ export interface LayoutNodeOptions {
   focused: boolean;
 }
 
-export interface LayoutLink extends NodesItemLinkInterface {
-  [key: string]: SafeAny;
+export interface LayoutLink extends NodesItemLink {
+  [key: string]: unknown;
 
-  detail: SafeAny;
+  detail: {
+    [key: string]: unknown;
+  };
```

Comment:
Isn't this just `Record<string, unknown>`?

Reply:
It is! Will change it.